### PR TITLE
Improve logging concept example

### PR DESCRIPTION
**Pull request by maltesander**

### CODE REVIEW

File: modules/concepts/pages/logging.adoc [hunk 1]
```diff
@@ -57,32 +57,45 @@ As part of the logging configuration, individual modules can be configured with
...
+    vectorAggregatorConfigMapName: vector-aggregator-discovery  // <1>
+  myRole:
+    config:                        // <2>
+      logging:
+        enableVectorAgent: true
+        containers:
+          main-container:
+            console:
+              level: INFO
+    myRoleGroup
```
Comment (sbernauer) [inline modules/concepts/pages/logging.adoc:71]:
```suggestion
    myRoleGroup:
```

**Review by sbernauer — APPROVED** [review]
LGTM otherwise

File: modules/concepts/pages/logging.adoc (hunk 1 above)
Comment (maltesander) [reply modules/concepts/pages/logging.adoc:71]: uh thanks!

**Review by sbernauer — APPROVED** [review]

**Comment by sbernauer** [comment]
Thanks!

**Pull request merged**